### DISCUSSION TITLE
how to Change Header Time to 24hr

### DISCUSSION THREAD

**Discussion by SweAdeel** [vkurko/calendar#356] [Q&A]
Hello
Thanks for this wonderful javascript ! really loved it
i have 3 questions

1- how do i change the time in header to 24hr ( image for ref)
![Screenshot_1](https://github.com/user-attachments/assets/30374e4c-bd19-4979-8a59-b658d137b46e)

2- how do i reduce the cell size  ( image for ref)
![Screenshot_2](https://github.com/user-attachments/assets/340d0c37-210d-4f2e-bdbe-1eb4ac7e64ca)

3- how to move calender to currenttime when open so we dont have to scroll all the way

waiting for your kind and helpful reply

cheers !

**Comment by vkurko** [comment]
Hello,

1. Please see [slotLabelFormat](https://github.com/vkurko/calendar?tab=readme-ov-file#slotlabelformat) option.
2. See [slotWidth](https://github.com/vkurko/calendar?tab=readme-ov-file#slotwidth).
3. See [scrollTime](https://github.com/vkurko/calendar?tab=readme-ov-file#scrolltime). However, in `timeline` views it does not scroll to the present day.

**Reply by SweAdeel** [reply]
thanks for reply

in point 3 , i want to highlight current time

can you give me a sample code for point 1 i.e slotLabelFormat

cheers !

**Reply by vkurko** [reply]
3. There is [nowIndicator](https://github.com/vkurko/calendar?tab=readme-ov-file#nowindicator) feature, but it currently does not woкk in timeline views.

1. `slotLabelFormat: {hour: 'numeric', minute: '2-digit', hour12: false}`
